docs/api_reference.rst

```diff
@@ -9,13 +9,40 @@ methods in the current release of PyMC experimental.
   :maxdepth: 2
 
 
+:mod:`pymc_experimental`
```

Comment:
`:mod:` should not be used for the headings. Used like this it is a cross-referencing role that links to the pymc_experimental module (that is, to the place where the `.. module:: pymc_experimental` or `.. automodule:: pymc_experimental` directive is, as this doesn't exist for now right now it does nothing and the cross-reference falls back to raw text, so we can use the raw text. Even if we added the directive it would basically be a link pointing basically to the same place.